packages/create-package/create-package.js

```diff
@@ -116,6 +116,10 @@ const generateFilesContent = (name, tag, typescript) => {
 		},
 	};
 
+	if (typescript) {
```

Comment:
it should be a `devDependency` for all projects, since the eslint plugin is depending on it even for plain packages

Reply:
As we talked it's removed and added as peer dependency